test/remote/gateways/remote_cyber_source_rest_test.rb

```diff
@@ -141,6 +199,56 @@ def test_failure_credit
     assert_equal 'INVALID_ACCOUNT', response.error_code
   end
 
+  def test_successful_authorize_with_apple_pay
+    response = @gateway.authorize(@amount, @apple_pay, @options)
+
+    assert_success response
+    assert response.test?
+    assert_equal 'AUTHORIZED', response.message
+    refute_empty response.params['_links']['capture']
+  end
+
+  def test_successful_authorize_with_google_pay
+    response = @gateway.authorize(@amount, @apple_pay, @options)
+
+    assert_success response
+    assert response.test?
+    assert_equal 'AUTHORIZED', response.message
+    refute_empty response.params['_links']['capture']
```

Comment:
What exactly are these assertions checking, and is there a reason they are present for these first two tests and not for the other new ones?

Reply:
For refute_empty, the reason is that I'm performing an authorization, and I want to be sure that an `authorization` with google_pay will contain (like any authorization) the capture link.
I think I can remove the `response.test?` part.